style.css

```diff
@@ -63,4 +63,372 @@
   .icons {
     padding-right: 1.5rem;
   }
+
+  .projects {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1rem;
+    gap: 0.75;
+    width: 20.438rem;
+    height: 34.5rem;
+    background: #fff;
+    border: 0.063rem solid #dfe1e6;
+    border-radius: 1rem;
+  }
+
+  .snapshots {
+    width: 18.438rem;
+    height: 13.75rem;
+    background-color: rgb(211, 143, 95);
+    border-radius: 0.5rem;
+  }
+
+  #snap2 {
+    background-color: #d0c7d1;
+  }
+
+  #snap3 {
+    background-color: rgb(231, 119, 44);
+  }
+
+  #snap4 {
+    background-color: rgb(255, 255, 255);
+  }
+
+  .project_name {
+    margin-top: 0.2rem;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 1200;
+    font-size: 1.8rem;
+    line-height: 2.75rem;
+    margin-bottom: -0.5rem;
+  }
+
+  .details {
+    display: inline-flex;
+    flex-direction: row;
+  }
+
+  .detail {
+    display: inline-flex;
+    color: #344563;
+    padding-right: 2.6rem;
+    margin-left: -2.3rem;
+  }
+
+  .projo {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .info_block {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0.75rem;
+    width: 18.438rem;
+    height: 18rem;
+  }
+
+  .info {
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 0.938rem;
+    line-height: 1.5rem;
+    color: #344563;
+    margin-top: -1rem;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 0.5rem;
+    width: 13.375;
+    height: 1.5rem;
+    margin-left: -13%;
+  }
+
+  .language {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 57px;
+    background: #ebebff;
+    border-radius: 8px;
+    height: 16px;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070fd;
+  }
+
+  #html {
+    width: 2.063rem;
+  }
+
+  #css {
+    width: 1.56rem;
+  }
+
+  #javascript {
+    width: 4.25rem;
+  }
+
+  .button {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 126px;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 15px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+  }
```

Comment:
- [ ] Ouch! this is close but as mentioned previously you're button should look similar to the given design you seem to be missing the background-color property in your implementation as it should be `#FFFFFF`
- Your design:
  ![image](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/104968236/923c266f-dad1-427f-a0bb-b3f4ad9e34b7)
- Figma design:
  ![image](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/104968236/6fe77057-572a-4428-8959-3a8020480b1b)

- [ ] [OPTIONAL] I'd also suggest you implement the `:active` & `:hover` state in your design as this increases your coverage of the original design
  ![firefox_7KIb3eS8J1](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/104968236/2152f9d2-c273-4e56-b65f-9c6b06ed4b15)